docs/monitoring-guidelines.md

```diff
@@ -9,7 +9,12 @@ The KubeVirt Users should have the same experience when searching for a node, co
 
 **Naming requirements:**
 1. Check if a similar Kubernetes metric, for node, container or pod, exists and try to align to it.
-2. KubeVirt metric for a running VM should have a `kubevirt_vmi_` prefix
+2. KubeVirt metrics prefixes:
+   1.  Running VM metrics should have a `kubevirt_vmi_` prefix
```

Comment:
If we do this, let's also already cover other components we have.
For example `api`. Our other components like `operator` could also have their respective prefix.
Tbh. I'm not entirely sure if metrics should really be prefixed with those details as the metric name should make sense without it and the metrics consumer should not necessarily care about the component emiting them, but the meaning they have.

Reply:
@kwiesmueller The intent of the prefix is to state for which operator it relates to. 
The operators metrics help with determining the operators health and performance.
Example: kubevirt_hco_out_of_band_modifications_count, which is specific to the hco operator.

Reply:
For metrics that exist across all our components I'd prefer them to have the same name and use different labels instead.
This makes them easier to query together.
Does that collide with this?

Reply:
@kwiesmueller No, it doesn't collide. They just need to have a kubevirt prefix.